lib/portage/tests/runTests.py

```diff
@@ -21,12 +21,7 @@ def debug_signal(signum, frame):
     pdb.set_trace()
 
 
-if platform.python_implementation() == "Jython":
```

Comment:
Tag the bug in your commit just for completeness

Reply:
Sure.